### ISSUE TITLE
elkjop.no: breakage

### ISSUE THREAD

**Issue by chrisco922** [uBlockOrigin/uAssets#12680]
### Prerequisites

- [X] I read and understood the [policy as to what is a valid filter issue](https://github.com/uBlockOrigin/uAssets/blob/master/README.md#uassets)
- [X] I verified that the issue has not already been reported (use [this button](https://user-images.githubusercontent.com/585534/146582579-c32707a0-36a1-4cc5-ad50-83172c9f67a8.png) to find out)
- [X] I forced an update of my filter lists (**how to do this:** [click _"Purge all caches"_](https://github.com/gorhill/uBlock/wiki/Dashboard:-Filter-lists#purge-all-caches), then [click _"Update now"_](https://github.com/gorhill/uBlock/wiki/Dashboard:-Filter-lists#update-now))
- [X] I did not remove any of the [default filter lists](https://user-images.githubusercontent.com/585534/91296476-ab9d8c00-e76a-11ea-9fb9-87d6861bdf73.png), or I verified that the issue is not caused by removing any of the default lists
- [X] I did not enable external filter lists, or I verified that the issue still occurs without enabling external filter lists
- [X] I do not have custom filters/rules, or I verified that the issue still occurs without custom filters/rules
- [X] I am not using [another content blocker along uBO](https://twitter.com/gorhill/status/1033706103782170625) (also mind browser built-in blockers)
- [ ] I did not answer truthfully to **all** the above checkpoints

### URL address of the web page

`https://www.elkjop.no/product/gaming/spillkonsoller-og-tilbehor/nintendo-switch/nintendo-konsoll/nintendo-switch-oled-gamingkonsoll-med-hvite-joy-con-kontroller-eu/428486?utm_medium=Pricecomparison`

### Category

breakage

### Describe the issue

When Ublock origin is enabled it blocks user from sending a product review. So the website tells me that anyway. I disabled all other addons just to verify.

### Screenshot(s)

<details>
<summary>Screenshot(s)</summary>

</details>

### Configuration

<details>

```yaml
uBlock Origin: 1.42.4
Chromium: 100
filterset (summary): 
  network: 81114
  cosmetic: 44101
  scriptlet: 16430
  html: 0
listset (total-discarded, last updated): 
  default: 
    user-filters: 0-0, never
    NOR-0: 2693-0, 5h.18m
    easylist: 65541-51, 2d.1h.1m
    easyprivacy: 26761-122, 2d.1h
    plowe-0: 3690-811, 10d.5h.43m
    ublock-abuse: 72-0, 2d.1h.3m
    ublock-badware: 4012-91, 2d.59m
    ublock-filters: 30450-93, 2d.58m
    ublock-privacy: 207-2, 2d.1h.2m
    ublock-unbreak: 1740-42, 2d.57m
    urlhaus-1: 7846-0, 5h.17m
filterset (user): [empty]
trustedset: 
  added: [array of 2 redacted]
switchRuleset: 
  added: [array of 1 redacted]
modifiedUserSettings: [none]
modifiedHiddenSettings: [none]
supportStats: 
  allReadyAfter: 1120 ms (selfie)
  maxAssetCacheWait: 94 ms
popupPanel: 
  blocked: 6
  blockedDetails: 
    bazaarvoice.com: 2
    dynamicyield.com: 1
    googletagmanager.com: 1
    iesnare.com: 1
    instana.io: 1
```
</details>

**Comment by stephenhawk8054** [comment]
Can you check if adding this filter works for you?

`*$script,redirect-rule=noopjs,domain=elkjop.no`

**Comment by krystian3w** [comment]
fyi @DandelionSprout / rare @THEtomaso and maybe @iam-py-test

**Comment by krystian3w** [comment]
> NOR-0: 2693-0, 5h.18m

fyi @DandelionSprout / rare @THEtomaso and maybe @iam-py-test

**Comment by krystian3w** [comment]
> NOR-0: 2693-0, 5h.18m

fyi @DandelionSprout / rare @THEtomaso and maybe @iam-py-test

**Comment by krystian3w** [comment]
> NOR-0: 2693-0, 5h.18m 

fyi @DandelionSprout / rare @THEtomaso and maybe @iam-py-test

**Comment by krystian3w** [comment]
> NOR-0: 2693-0, 5h.18m

fyi @DandelionSprout / rare @THEtomaso and maybe @iam-py-test

**Comment by DandelionSprout** [comment]
I did not experience any problems with at least *submitting* a review for `https://www.elkjop.no/product/gaming/spill/kirby-and-the-forgotten-land-switch/410327#reviews` with uBO in Chrome Manifest V2. 

<details>

![image](https://user-images.githubusercontent.com/22780683/162619462-70f9c601-fc9a-4671-993c-c409eb014c9d.png)
</details>

**Comment by stephenhawk8054** [comment]
> I did not experience any problems with at least _submitting_ a review for `https://www.elkjop.no/product/gaming/spill/kirby-and-the-forgotten-land-switch/410327#reviews` with uBO in Chrome Manifest V2.

Mine shows this line: `Du må deaktivere reklameblokkerere fordi det kan hindre innsendelse av anmeldelsen.
`

<details><summary>Screenshot 1</summary>

![image](https://user-images.githubusercontent.com/66517106/162620246-7b744b30-9948-41c5-a910-cbbfdf9f7a4f.png)

</details>

I'm not sure if it's geo-related or not, but looks like I got different blocked logs:

<details><summary>Screenshot 2</summary>

![image](https://user-images.githubusercontent.com/66517106/162620308-ee5a41ba-beaa-4e1f-a9e3-f64714eb8f81.png)

</details>

I tested with OP's configuration btw: uBO's default + `NOR, DNK, ISL: Dandelion Sprouts nordiske filtre`

**Comment by mapx-** [comment]
@DandelionSprout 
go to 
`https://www.elkjop.no/product/gaming/spillkonsoller-og-tilbehor/nintendo-switch/nintendo-konsoll/nintendo-switch-oled-gamingkonsoll-med-hvite-joy-con-kontroller-eu/428486?utm_medium=Pricecomparison#reviews`

go down where is `Hva andre tenker om produktet`

click the button on the right side `Skriv en produktanmeldelse` 

You'll get:

![image](https://user-images.githubusercontent.com/10303732/162620407-479bf42c-0ab2-4ab0-b8bb-db735752ca28.png)

**Comment by DandelionSprout** [comment]
My core setup differs somewhat from the one recommended by the uBO team, as I always turn on AdGuard Base and turn off EasyPrivacy.

I am 115% sure that Nordic Filters are not to blame, and would've bet actual money on EasyPrivacy being to blame.

**Comment by mapx-** [comment]
@stephenhawk8054 
it seems to be due to blocking (Peter Lowe's `||iesnare.com^` )
`https://mpsnare.iesnare.com/snare.js)https://mpsnare.iesnare.com/snare.js`

**Comment by mapx-** [comment]
However, your filter seems to work

**Comment by stephenhawk8054** [comment]
> However, your filter seems to work

Yup, I was just waiting for OP to confirm about the final step (submission), as I couldn't test that part. But hope it still works fine.

**Comment by THEtomaso** [comment]
I've actually noticed long ago that user reviews at `elkjop.no` won't display at all, but I just haven't bothered to report it.
..but, since the issue now has been brought up:

### Problem filters:
* **AdGuard Annoyances filter**
`###CookieConsent`
`||policy*.cookieinformation.com^$third-party`

* **EasyList Cookie List** / **Fanboy's Annoyance List**
`###coiOverlay[role="banner"]`
`###CookieConsent`
`||cookieinformation.com/*/*.js`

* **I don't care about cookies** _(@kiboke)_
`###Coi-Renew`
`###CookieConsent:not(.modal)`
`policy.app.cookieinformation.com$domain=~kartor.eniro.se|~kart.gulesider.no|~kort.degulesider.dk|~map.krak.dk|~altibox.no|~babysam.dk|~minaftale.dk|~skousen.dk|~skousen.no|~whiteaway.com|~whiteaway.no|~whiteaway.se|~dvdoo.dk`

### Multi-fix:
```
elkjop.no#@###Coi-Renew
elkjop.no#@##coiOverlay[role="banner"]
elkjop.no#@##CookieConsent
elkjop.no#@##CookieConsent:not(.modal)
@@||policy.app.cookieinformation.com^$domain=elkjop.no
```

**Comment by mapx-** [comment]
@kiboke @ryanbr @AdamWr

**Comment by Yuki2718** [comment]
@DandelionSprout @THEtomaso Can you check if the issue is fixed for AG?

**Comment by kiboke** [comment]
@THEtomaso thanks, it will be fixed in IDCAC.
Btw, I don't think `#CookieConsent:not(.modal)` is a problematic selector since it only matches a `<script>`.

**Comment by hdbordercollie** [comment]
Cookie banner popup now, example page: https://www.elkjop.no/product/mobil-nettbrett-og-smartklokker/mobiltelefon/samsung-galaxy-s24-ultra-5g-smarttelefon-12256gb-titanium-black/727287

**Comment by derpina-ai** [comment]
constant reload of page on any menu now with ublock on

**Comment by mapx-** [comment]
@derpina-ai 
- provide all the necessary steps to reproduce the issue
- provide your configuration info (browser, uBO or uBOL ?, filter lists used, etc)
- disable any other extension while testing
